modules/battery_manager/test/TestPlan.md

```diff
@@ -67,7 +67,6 @@ Test Battery.
 
 **Testing Change Status column**
   1. Press the `Deactivate` button in the `Change Status` column on an entry in the data table.
-     - Ensure that a warning message appears that asks you to confirm the action.
```

Comment:
@HenriRabalais isnt there supposed to be a warning ?

Reply:
It's not working anymore if it is.

Reply:
Umm, I don't there is for the Deactivate/Activate button.